cuberite/README.md

```diff
@@ -0,0 +1,25 @@
+# Cuberite snap
+
+This project creates a working snap of Cuberite from upstream git.
+
+To get this done, we need to do the following:
+ - build from current tip of git.
+ - stage the Server directory from the source tree.
+ - Add a wrapper to copy runtime files to a writable directory and launch the
+   server from there.
+
+## Current state
+
+Working features:
+ - Game tested with Minecraft 1.9 client.
+ - Web Admin.
+ - Plugins.
+
+Known issues:
+  - Still needs devmode thanks to a call to fchown in the statically-linked
+    sqlite3.
+
```

Comment:
On Tuesday, 9 August 2016 07:44:00 BST, Daniel Holbach 
<notifications@github.com> wrote:
>> + - build from current tip of git.
>> + - stage the Server directory from the source tree.
>> + - Add a wrapper to copy runtime files to a writable 
>> directory and launch the
>> +   server from there.
>> +
>> +## Current state
>> +
>> +Working features:
>> + - Game tested with Minecraft 1.9 client.
>> + - Web Admin.
>> + - Plugins.
>> +
>> +Known issues:
>> +  - Still needs devmode thanks to a call to fchown in the 
>> statically-linked
>> +    sqlite3.
>> +
> 
> This PR seems to change things from `devmode` to `strict` though...

Argh! Good catch, thanks. The devmode is still required, I was testing it 
in strict and must have forgotten to change it back.


-- 
Sent using Dekko from my Ubuntu device